src/index.html

```diff
@@ -4,11 +4,22 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>To Do List</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
     <br>
-    <div id="data">
-        <h1> to do list</h1>
+    <div class="container">
```

Comment:
- Optional
Consider fixing the overlapping issue of the save button.
![image](https://user-images.githubusercontent.com/33268927/179160891-a72eb0dc-ec62-410c-98b3-ce84bb25b2cb.png)